auth-web/src/components/auth/common/AccountBusinessType.vue

```diff
@@ -281,7 +280,7 @@ export default class AccountBusinessType extends Vue {
   }
 
   private get enableOrgNameAutoComplete (): boolean {
-    return LaunchDarklyService.getFlag(LDFlags.EnableOrgNameAutoComplete) || false
+    return true
```

Comment:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/3996d9a8-1ef3-4cd9-a599-09e6d7e4e5a2)

Reply:
Instead of a trivial getter, why not resolve where it's used in here (3 places).

Reply:
Thanks. Looks good.